src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: inherit;
+  font-size: inherit;
+  line-height: inherit;
+  color: inherit;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.img {
+  width: 40px;
+  height: 40px;
+  margin-right: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-sizing: border-box;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+.nav_list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  list-style: none;
+  padding: 0;
+  height: 60px;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.navigation a {
+  display: block;
+  position: relative;
+  text-decoration: none;
+}
```

Comment:
Don't use tag names for styling

Reply:
isn't fixed all over the page (pay attention here `.is-active, a:hover `)